book/src/remapping.md

```diff
@@ -53,4 +53,5 @@ Control, Shift and Alt modifiers are encoded respectively with the prefixes
 
 Keys can be disabled by binding them to the `no_op` command.
 
-Commands can be found in the source code at [`helix-term/src/commands.rs`](https://github.com/helix-editor/helix/blob/master/helix-term/src/commands.rs)
+Commands can be found at [Keymap](https://docs.helix-editor.com/keymap.html) Commands.(Right pane)
```

Comment:
`Commands.(Right pane)` at the end ?

Reply:
I'm sorry, didn't get you.

I put it there just to make sure the people use the `Commands` pane, which is the right side.

Reply:
Ah you meant the right column, I don't think we need to mention it since it's pretty self evident.

Reply:
👍 I'll remove it.

Reply:
Done 👍